src/sentry/grouping/enhancer/enhancement-configs/newstyle@2023-01-11.txt

```diff
@@ -165,7 +165,6 @@ family:native function:_pthread_start category=threadbase
 family:native function:__pthread_start category=threadbase
 family:native function:_pthread_body category=threadbase
 family:native function:_dispatch_worker_thread2 category=threadbase
-family:native function:_dispatch_client_callout category=threadbase
```

Comment:
Being marked as `threadbase` applies the following rule: `category:threadbase -group v-group`